block-producer/src/batch_builder/mod.rs

```diff
@@ -17,7 +17,8 @@ pub(crate) const CREATED_NOTES_SMT_DEPTH: u8 = 13;
 /// The created notes tree uses an extra depth to store the 2 components of `NoteEnvelope`.
 /// That is, conceptually, notes sit at depth 12; where in reality, depth 12 contains the
 /// hash of level 13, where both the `note_hash()` and metadata are stored (one per node).
-const MAX_NUM_CREATED_NOTES_PER_BATCH: usize = 2usize.pow((CREATED_NOTES_SMT_DEPTH - 1) as u32);
+pub(crate) const MAX_NUM_CREATED_NOTES_PER_BATCH: usize =
+    2usize.pow((CREATED_NOTES_SMT_DEPTH - 1) as u32);
```

Comment:
tiny nit: we usually write `2_usize` vs. `2usize`.

Reply:
done